csrankings-v.csv

```diff
@@ -168,6 +168,8 @@ Vernon Rego,Purdue University,https://www.cs.purdue.edu/people/rego,NOSCHOLARPAG
 Vero Vanden Abeele,KU Leuven,https://iiw.kuleuven.be/onderzoek/emedia/people/verovandenabeele,4ujvHaEAAAAJ
 Verónica Becher,University of Buenos Aires,http://www-2.dc.uba.ar/profesores/becher,_7APZ6sAAAAJ
 Verónica Bolón,University of A Coruña,https://pdi.udc.es/en/File/Pdi/B93TH,ameK2ocAAAAJ
+Veronica Catete,North Carolina State University,https://www.csc.ncsu.edu/people/vmcatete,TxIMXZoAAAAJ

```

Comment:
This entry (without accents) does not correspond to a DBLP alias. Please remove.

Reply:
entry removed. Thank you.